app/styles/libs/_helpers.scss

```diff
@@ -104,10 +104,6 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   white-space: nowrap;
 }
 
-.pre-line {
```

Comment:
Why?

Reply:
Sir, the Reason is it was giving extra space as you can see in the screenshot given by Mario Sir, And sir told me to remove those extra spaces and make it look like the screenshot given earlier by me.

Reply:
So you'll remove the class? Imagine if doctors remove the heart of the patient if they have heart cancer

Reply:
So sorry sir, I will undo the changes and will try to do it without deleting the classes. Once again I am so sorry for such thing.

Reply:
The class is still removed